SUMMARY.md

```diff
@@ -2,81 +2,107 @@
+  * [Browsing a Directory of Pieces](tutorials/core-concepts/03-reusable-content-pieces/browsing-directory-of-pieces.md)
+  * [Joins: Connecting Piece Types to One Another](tutorials/core-concepts/reusable-content-pieces/children-and-joins.md)
+* [Global Settings](tutorials/core-concepts/04-global-settings/README.md)
+  * [The global doc: sharing content across pages](tutorials/core-concepts/04-global-settings/global.md)
+  * [Global Settings](tutorials/core-concepts/04-global-settings/settings.md)
+* [Modules](tutorials/core-concepts/06-modules/README.md)
+  * [How Apostrophe Modules are Structured](tutorials/core-concepts/06-modules/how-apostrophe-modules-are-structured.md)
+  * [Nested Module Folders](tutorials/core-concepts/06-modules/nested-module-folders.md)
+  * [More Modules](tutorials/core-concepts/06-modules/more-modules.md)
+* [Front End Assets](tutorials/core-concepts/07-front-end-assets/README.md)
+  * [Pushing assets to the browser](tutorials/core-concepts/07-front-end-assets/pushing-assets.md)
+  * [Lean frontend assets: Apostrophe without jQuery](tutorials/core-concepts/07-front-end-assets/lean-frontend-assets.md)
+* [Users and Permissions](tutorials/core-concepts/08-users-and-permissions/README.md)
+  * [Users and Groups](tutorials/core-concepts/08-users-and-permissions/users-and-groups.md)
+  * [Managing Access Control](tutorials/core-concepts/08-users-and-permissions/managing-access-control.md)
+  * [User Login and Password Help](tutorials/core-concepts/08-users-and-permissions/user-login-and-password.md)
+* [Working With Templates](tutorials/core-concepts/09-working-with-templates/README.md)
+  * [Nunjucks Filters](tutorials/core-concepts/09-working-with-templates/nunjucks-filters.md)
+  * [Nunjucks Helper Functions](tutorials/core-concepts/09-working-with-templates/nunjucks-helper-functions.md) 
+  * [Accessing images inside related pages](tutorials/core-concepts/09-working-with-templates/thumbnails-and-areas-of-child-pages.md)
+  * [Responsive Images](tutorials/core-concepts/09-working-with-templates/responsive-images.md)
+* [Apostrophe Search](tutorials/core-concepts/10-apostrophe-search/README.md)
+  * [Enabling Search](tutorials/core-concepts/10-apostrophe-search/search.md)
 
+## Advanced Topics
+* [Advanced Pieces Topics](tutorials/advanced-development/01-advanced-pieces-topics/README.md)
+  * [AJAX Features: Enhanced Browsing for Pieces](tutorials/advanced-development/01-advanced-pieces-topics/01-ajax-pieces.md)
+  * [Fine-grained Permissions for Pieces](tutorials/advanced-development/01-advanced-pieces-topics/02-permissions-for-pieces.md)
+  * [Adding Filters](tutorials/advanced-development/01-advanced-pieces-topics/04-adding-filters.md)
+  * [Adding Columns](tutorials/advanced-development/01-advanced-pieces-topics/05-adding-columns.md)
+  * [Using the Pieces Index](tutorials/advanced-development/01-advanced-pieces-topics/06-next-previous-links.md)
```

Comment:
This page doesn't really work and should be broken up, see my notes on the individual file.